test/extended/jenkins/kubernetes_plugin.go

```diff
@@ -83,9 +83,15 @@ var _ = g.Describe("[jenkins] schedule jobs on pod slaves", func() {
 			o.Expect(err).NotTo(o.HaveOccurred())
 			o.Expect(port).NotTo(o.BeEmpty())
 
+			j := JenkinsRef{
+				oc:   oc,
+				host: serviceIP,
+				port: port,
+			}
+
 			jenkinsUri := fmt.Sprintf("http://%s:%s", serviceIP, port)
 			g.By(fmt.Sprintf("wait for jenkins to come up at %q", jenkinsUri))
```

Comment:
so is the `jenkinsUri` just debug at this point?  If so, we don't need to bother with constructing an http url string.  Just print the ip and port in the `g.By`